enhancements/oc/must-gather.md

```diff
@@ -74,6 +74,10 @@ ship a second one.
    and only contains logic to gather information from that payload. We have e2e tests that make sure this functions.
 3. `oc adm must-gather --image` which is a client-side tool that runs any must-gather compatible image by creating a pod,
    running the `/usr/bin/gather` binary, and then rsyncing the `/must-gather` and includes the logs of the pod.
+4. `oc adm must-gather --all-images` allows the user to perform the same task as `oc adm must-gather --image`, but 
```

Comment:
Good to see that movement and thanks for the PR. 

Currently we are trying to limit the volume size to protect core components in master node and I think, this feature will drastically increase the risk of filling storage. I'd prefer adding this feature after volume size limiting https://github.com/openshift/oc/pull/1533 is completed.

Reply:
xref: https://issues.redhat.com/browse/RFE-4568

Reply:
@soltysh can https://issues.redhat.com/browse/RFE-1576 also be worked into this preposal).

Reply:
@ardaguclu I understand the point about volume size limiting. But do you think we can get this proposal PR merged? As for the code PR, I shall open one after https://github.com/openshift/oc/pull/1533 is merged. WDYT?

Reply:
I'm fine with merging this proposal because it sounds reasonable to me.

Reply:
> @soltysh can https://issues.redhat.com/browse/RFE-1576 also be worked into this preposal).

Based on my comment in that RFE it was deferred, so I'd prefer we don't.